roles/reproducer/tasks/configure_cleanup.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Configure cleanup
+  delegate_to: controller-0
+  block:
+    - name: Discover and expose CI Framework path on remote node
+      tags:
+        - always
+      vars:
+        default_path: >-
+          {{
+            cifmw_reproducer_default_repositories |
+            selectattr('src', 'match', '^.*/ci[_\-]framework$') |
+            map(attribute='dest') | first
+          }}
+        custom_path: >-
+          {{
+            cifmw_reproducer_repositories |
+            selectattr('src', 'match', '^.*/ci-framework$') |
+            map(attribute='dest')
+          }}
+        _path: >-
+          {{
+            (custom_path | length > 0) |
+            ternary(custom_path | first, default_path)
+          }}
+      ansible.builtin.set_fact:
```

Comment:
I'm not following what you're trying to do here. You're discovering `default_path`, then you're trying to fetch the `custom_path` which is only different by `_-`. If it's not found, you're reverting to the `default_path`. Is that correct?
Why is it like that?

Reply:
This is similar to the configure architecture TBH didn't change this part of the logic but maybe we can reconsider it in both

Reply:
Ack. Let's keep the same logic. We can change that later.